R-graphql/README.md

```diff
@@ -0,0 +1,54 @@
+## Santiment API R example
+
+### Usage
+
+```
+SANBASE_API_KEY=put-your-sanbase-apikey Rscript example.r
+```
+
+### Getting Apikey
+
+To obtain an api key you should [log in to sanbase](https://sanbase-low.santiment.net/login) and go to the `account` page - [https://sanbase-low.santiment.net/account](https://sanbase-low.santiment.net/account). There is an `API Keys` section and a `Generate new api key` button.
```

Comment:
I think we should mention that you don't need an API key, but you will get limited data.

Reply:
Added some explanation